src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -76,6 +76,20 @@ export class UpperBarManager {
     return icons.sort((a, b) => iconsOrder[a.displayName] - iconsOrder[b.displayName]);
   }
 
+  public focusPluginButton(pluginName: string) {
+    let pluginButton;
+    const controls = this.getControls(this.iconsOrder);
+    const pluginElement = controls.find((control) => control.displayName === pluginName)?.componentRef?.current
```

Comment:
can we use plugin button id (https://github.com/kaltura/playkit-js-ui-managers/blob/master/src/services/upper-bar-manager/models/icon-model.ts#L10) instead of label?

Reply:
done